less/home-page.less

```diff
@@ -1,75 +1,152 @@
 // Home Content
 
 .home {
-
+  width: 900px;
+  box-sizing: border-box;
   .intro {
 
     h2 {
       padding: 15px;
+      .text-shadow();
     }
 
     p {
       padding:0 15px;
+
+   @media @phone {
+     width: 475px;
+   }
     }
   
   }
 
   .space-images {
     display: flex;
     flex-wrap: wrap;
-    width: 100%;
-
+    width: 850px;
+    @media @phone {
```

Comment:
I suggest doing some self study on media queries this weekend. Maybe some coding challenges to do with media queries as well!